### PR TITLE
Add `starts_with` and `convert_prefix`

### PR DESCRIPTION
**Pull request by KTamas**
We needed a way to pull down `n` secrets to the environment that has a common prefix, so I added that feature as well as another one that does not apply string conversions on said prefix. So the secret `TF_VAR_AWS_ACCESS_KEY` becomes `TF_VAR_aws_access_key` and so on.

I've also updated to the latest versions of dependencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # secrets-to-env
 
 <p align="center">
-  <a href="https://github.com/oNaiPs/secrets-to-env-action/actions"><img alt="secrets-to-env-action status" src="https://github.com/oNaiPs/secrets-to-env-action/workflows/build-test/badge.svg"></a>
+  <a href="https://github.com/compasspathways/secrets-to-env-action/actions"><img alt="secrets-to-env-action status" src="https://github.com/compasspathways/secrets-to-env-action/workflows/build-test/badge.svg"></a>
```
Comment (oNaiPs) [inline README.md:4]: Please update back to original

File: package.json [hunk 2]
```diff
@@ -25,23 +25,23 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "^1.9.0",
+    "@actions/core": "^1.10.0",
```
Comment (oNaiPs) [inline package.json:28]: Remove dependency updates from this PR

**Review by oNaiPs — CHANGES_REQUESTED** [review]
Many thanks for your contribution! Please check the comments I left, and:
- Please remove the distribution files, I will generate them after the PR is merged

**Comment by oNaiPs** [comment]
Integrated into `main`. Thanks!

**Comment by KTamas** [comment]
Thanks, apologies for not getting around to make the proposed changes.

**Pull request closed without merging**